specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2020-09-01-preview/iotdps.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "2020-09-01-preview",
+    "version": "2020-09-01-preview.1",
```

Comment:
Is it valid to change version like this?
cc: @akning-ms

Reply:
I was given f/b that this is the guideline in order to make changes to an API that is still in development. I am checking with the approver and sharing the failure seen below.

Reply:
I've removed this version change since I added more changes and wanted the validations to re-run. I do not have clarity on what direction to proceed since the tests fail if this requirement were to be enforced.